documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -65,6 +65,12 @@ Operator 2.5.0+ is certified for use on OpenShift Container Platform 4.3.0+ with
 When using the operator in OpenShift, a security context constraint is required to ensure that WebLogic containers run with a UNIX UID that has the correct permissions on the domain file system.
 This could be either the `anyuid` SCC or a custom one that you define for user/group `1000`. For more information, see [OpenShift]({{<relref "/security/openshift.md">}}) in the Security section.
 
+Beginning with operator version 4.0, you should specify the `kubernetesPlatorm` Helm chart property and set its value to `Openshift` when installing the operator in Openshift. With this setting, the operator:
+- Configures the correct file permissions for WebLogic Server to work in Openshift for [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), and [Domain home in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) domains.
+- Sets `weblogic.SecureMode.WarnOnInsecureFileSystem` Java system property to `false` on each target WebLogic Server instance. This flag suppresses the insecure file system warnings in the WebLogic Server console in production mode. These warnings result from setting the correct file permissions to work with restricted security context constraints on Openshift.
+For more information about the Helm chart, see the
+[Operator Helm configuration values]({{<relref "/userguide/managing-operators/using-helm#operator-helm-configuration-values">}}).
```

Comment:
Are extra steps needed for Domain in PV (I'm wondering if the user may need to run a chmod -R from within a container after creating the domain home...)?  If so, they should be documented here.

Reply:
I'm not sure, I can't run the Domain in PV test in Openshift environment. I checked with Bhavani and she's also having trouble running any of the Domain in PV tests on OKD.

Reply:
I was able to successfully run the Domain in PV in the PKD environment with and without `kubernetesPlatform=Openshift` flag. The test passed in both cases, it looks like no extra steps are needed for Domain in PV.

https://build.weblogick8s.org:8443/job/wko-okd-test/16/ (with `kubernetesPlatform=Openshift`)
https://build.weblogick8s.org:8443/job/wko-okd-test/14/ (without `kubernetesPlatform` flag)